### DISCUSSION TITLE
一人血书harmony os next 版本

### DISCUSSION THREAD

**Discussion by ThePonyZ** [amll-dev/applemusic-like-lyrics#265] [General]
不知大佬是否会ArkTS

**Comment by Steve-xmh** [comment]
有以下原因，时机未成不考虑：
- HarmonyOS NEXT 同样允许使用浏览器视图控件，一样可以呈现 AMLL 的歌词组件
- 作为新兴运行环境群体，正式版本尚未推行，我们这些小众软件并没有强烈动机去为这些平台提供支持（参考 Windows Phone）
- DevEco Studio 的 SDK 和部分工具链的舒适程度远不如作者直接使用 VSCode 配合各大扩展来的便利，且 SDK 的不透明的地方太多，不利于深度学习开发
- 作者能力有限，只希望尽量花充足精力去打磨好 AMLL 在浏览器上的表现水平

**Comment by Linho1219** [comment]
暂无相关开发计划。感谢理解。
